### PR TITLE
Add Unsplash selector

### PR DESCRIPTION
**Pull request by ChristopherJennings**
### Motivation

Adds a new image selector using the popular Unsplash website as a source.

### Checklist

- [x] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [x] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?
Follow set up instructions and use it 😉

### CODE REVIEW

**Comment by makma** [comment]
From my point of view, it's OK to put your Unsplash API key (or any other data) into properties JSON if both of the following conditions are fulfilled:

- Sensitive data could be exposed to the users of the particular Kentico Cloud project (eg. content editor) - in this case, this is true.
_For example, exposing content management API key for reviewer might be risky, since they could escalate their privileges using management API, thus content management API key shouldn't be stored in properties JSON._
- One has to consider, the key (or any data put in properties JSON) will be stored within Kentico Cloud infrastructure and might be also logged in 3rd party services according to [Terms of Use](https://kenticocloud.com/terms-of-service) - this might be OK as well. 
_As an example - storing API key for accessing medical DB is not a good idea at all._

In conclusion, as I see it, it's perfectly OK to put Unsplash key into properties JSON.

**Comment by ChristopherJennings** [comment]
I've addressed the concerns that were mentioned. Sorry, about the URL path, totally missed that. I've left the favicon it's just an artifact of the starter project and not really worth the effort of removing.

**Comment by ChristopherJennings** [comment]
@Simply007, should be taken care of now.
